app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue

```diff
@@ -184,13 +186,18 @@ export default {
       }));
     },
     addTagValue(tagValue) {
-      const tag = {
-        name: tagValue,
-      };
-      this.metaTags.push(tag);
-      this.$refs.tagInput.$el.focus();
+      const tags = tagValue.split(',').map(value => ({
+        name: value.trim(),
+      }));
+      this.metaTags.push(...tags);
       this.saveArticle();
     },
+    onBlur() {
+      const currentSearch = this.$refs.tagInput.$refs.search.value;
```

Comment:
This looks a bit hacky, but the `vue-multiselect` is not the ideal candidate for this kind of component with arbitrary tags and no list to select form. 

An alternative approach is that we could listen to the `@search-change` event. And store the result in a variable, onBlur we just assign the value to `currentSearch` and then reset the variable.

This is cleaner and does not leak the implementation details of the underlying component

Reply:
@scmmishra Thanks for the suggestion, it works fine.